test/unit/links.test.js

```diff
@@ -6,18 +6,21 @@ const assert = require('assert');
 const htmls = fixtures.readJSON('op_html.json');
 
 const expected = [{
-  fixes: ['https://github.com/node/issues/16437'],
+  fixes: ['https://github.com/nodejs/node/issues/16437'],
```

Comment:
I believe this test was incorrect before. Not sure whether I am supposed to pass `nodejs/node` as the repo name.

Reply:
I think we can just pass the repo + the owner name to `LinkParser`

Reply:
^to be clear, I meant change the signature of  the constructor of `LinkParser` to `(owner, repo, html)`